src/parser/parser.ts

```diff
@@ -10,8 +10,16 @@ import { globSync } from 'glob';
 export type Renderer = (content: string) => string;
 
 const pathHeading: Renderer = (path: string) => `# \`${path.replace(homedir(), '~')}\``;
-const wrap = (contentType: string, content: string) =>
-    `<div class="content-${contentType}">${content}</div>`;
+
+function wrap(contentType: string, content: string, linkPath?: string): string {
+    let link = '';
+    if (typeof linkPath !== 'undefined') {
```

Comment:
LMAO, I knew you'd pick that up.

I nearly made it a ternary on one one line reaching out to column 200 or something.

So what the best way to do this, there is a commit button here, or is it better if I make the change locally, test it and push up a new commit?

Reply:
> LMAO, I knew you'd pick that up.
> 
> I nearly made it a ternary on one one line reaching out to column 200 or something.

Hahahaha nice I actually love to do stuff like that😂 But here I try to hold back because I want it to be accessible to others

> So what the best way to do this, there is a commit button here, or is it better if I make the change locally, test it and push up a new commit?

The latter ^^

Reply:
I noticed, I seriously considered it, and I used just the shortened if matching against the empty string but removed it for readability. Don't know why I then didn't think to use that method on the expanded if statement.

If you don't mind me rewriting history I can squish this into the original commit and force push it up.

Reply:
> If you don't mind me rewriting history I can squish this into the original commit and force push it up.

No strong opinion, do as you please :)